ignite/distributed/launcher.py

```diff
@@ -15,7 +15,7 @@ class Parallel:
 
     - XLA on TPUs via `pytorch/xla <https://github.com/pytorch/xla>`_ (if installed)
 
-    - using `Horovod distributed framework <https://horovod.readthedocs.io>`_ (if installed)
+    - using `Horovod distributed framework <https://horovod.readthedocs.io>`__ (if installed)
```

Comment:
Here too

Reply:
Here too. There is 2 Horovod links (this one and another one below which i didn't edit)

Reply:
OK, let's use this one as named hyperlink and below one as anonymous.